qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +66,27 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
+            'MappingDirectoryFormat': MappingDirectoryFormat,
+            'IntSequenceV2DirectoryFormat': IntSequenceV2DirectoryFormat,
+            'IntSequenceFormatV2': IntSequenceFormatV2,
+            'FourIntsDirectoryFormat': FourIntsDirectoryFormat,
+            'IntSequenceFormat': IntSequenceFormat
+        }
+
+        self.assertEqual(obs, exp)
+
+    def test_importable_formats_with_dummy(self):
+        exp = "DummyFormat"
```

Comment:
`exp` is an odd name here because it isn't really the "expected" result (it's just a format name). Can you remove this variable and just use the format name in the assertion below? `obs` can also be removed and the assertion just becomes `self.assertFalse('DummyFormat' in self.pm.importable_formats)`